io-native/src/test/java/org/ballerinalang/stdlib/io/characters/AsyncReadWriteTest.java

```diff
@@ -44,7 +44,7 @@ public void setup() {
         currentDirectoryPath = System.getProperty("user.dir") + "/build/";
     }
 
-    @Test(description = "Tests reading characters through the async io framework")
+    @Test(enabled = false, description = "Tests reading characters through the async io framework")
```

Comment:
Why do we disable all the tests?

Reply:
We have to fix this Windows build https://github.com/ballerina-platform/module-ballerina-io/issues/4

Reply:
We can only disable in windows build right? ./gradlew clean build -x test

Reply:
IMO also, better to disable in windows build only for now. Running these tests even in a single environment would be better than skipping in both.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/5/commits/3e367134f7fd716ba32095be6a93ee260149df62